setup.py

```diff
@@ -47,7 +61,11 @@
     include_package_data=True,
     entry_points={
         'console_scripts': [
-            'axondeepseg = AxonDeepSeg.segment:main','axondeepseg_test = AxonDeepSeg.integrity_test:integrity_test'
+           'axondeepseg_models = AxonDeepSeg.models.download_model:main', 'axondeepseg = AxonDeepSeg.segment:main','axondeepseg_test = AxonDeepSeg.integrity_test:integrity_test'
```

Comment:
maybe change `axondeepseg_models` for `download_models`

Reply:
done in commit (b4a3276)